Makefile

```diff
@@ -55,7 +55,7 @@ install: install-k8s ## Install all rukpak core CRDs and provisioners
 
 run-local: install-apis ## Install CRDs and run provisioner locally
 	kubectl create namespace rukpak-system
-	$(Q)go run provisioner/k8s/main.go
+	nohup $(Q)go run provisioner/k8s/main.go &
```

Comment:
looked up `nohup`, very cool command!

Reply:
Yep I think I'm going to move that nohup call to the actual workflow file as I'd imagine run-local should run the provisioner in the foreground by default. This is just a hack until we get a better idea of the provisioner manifests + a more concrete CI pipeline.